data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,425 @@
+				decline
+
+	on complete
+		payment 300000
+		conversation
+			`You are surprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill in what must be a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for bringing our family back together" and gives you your payment of <payment>.`
+
+
+
+mission "Unfettered: Hunger Stories 2"
+	minor
+	name `Transport a Hai warrior to <destination>`
+	description `Transport an Unfettered warrior to <destination>.`
+	source "Darkcloak"
+	destination "Makerplace"
+	passengers 1
+	to offer
+		has "unfettered: hunger"
+		random < 40
+	on offer
+		conversation
+			`"Human!" The cry catches your attention not only because of the volume of the shout, but primarily because it's the most respectful term anyone has used for you in Unfettered space. You turn and see a young Hai warrior striding toward you. At least you think he's young. He's certainly thinner than most of the Hai you see south of Wah-Ki. You notice that his boots seems to be made more of mending tape than actual fabric. He's not wearing the ornately decorated knives that many of the True Hai carry, but he does have sheaths for them. Instead, he carries a long staff that looks to be made of one of the light but rigid Hai alloys. You can see that scars from many fights disturb the pattern of his fur.`
+			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasizing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travelers give him room.`
+			`	It takes you a moment to parse what he is saying as his accent is difficult to understand. You would guess he learned your language in class but has never spoken with humans. You think he is offering to be your translator and bodyguard.`
+			choice
+				`(Hire him.)`
+					goto hire
+				`(Politely decline.)`
+					goto polite
+				`(Laugh at him and walk away.)`
+					goto laugh
+			
+			label hire
+			`	The young Unfettered proves diligent in his work. He steers you away from some vendors and towards others and the prices and quality are at least competitive. Several times his staff has flashed out and blocked people who might have bumped into you. While some of the vendors you traded with were amused by his translations, most worked through him rather than speaking to you directly. He even kept watch while you were eating, though he did seem to be distracted by the food.`
```

Comment:
```suggestion
			`	The young Unfettered proves diligent in his work. He steers you away from some vendors and towards others where the prices and quality are at least competitive. Several times his staff has flashed out and blocked people who might have bumped into you. While some of the vendors you traded with were amused by his translations, most worked through him rather than speaking to you directly. He even kept watch while you were eating, though he did seem to be distracted by the food.`
```